pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4836,6 +4838,21 @@ static __always_inline bool skb_revalidate_data(struct __sk_buff *skb, uint8_t *
     return true;
 }
 
+void set_net_event_id(net_packet_t *pkt){
+    if (pkt->protocol == IPPROTO_UDP && pkt->dst_port == 53){
+        pkt->event_id = DNS_REQUEST;
+    }
+    if (pkt->protocol == IPPROTO_UDP && pkt->src_port == 53){
+        pkt->event_id = DNS_RESPONSE;
+    }
+}
+
+bool should_send_payload(net_packet_t *pkt){
```

Comment:
I think that the word we use to describe sending data from kernel to user is "submit" so I think you should call this like this.